src/main/java/io/github/debuggyteam/architecture_extensions/api/BlockType.java

```diff
@@ -61,14 +58,19 @@ public String toString() {
 		return name().toLowerCase(Locale.ROOT);
 	}
 
-	public TypedGroupedBlock register(BlockGroup group, BlockGroup.GroupedBlock groupedBlock, ArchExIntegrationContextImpl.BlockCreationCallback onBlockCreated) {
-		var id = ArchitectureExtensions.id(groupedBlock.id().getPath() + "_" + this);
+	public TypedGroupedBlock register(BlockGroup group, BlockGroup.GroupedBlock groupedBlock, ArchExIntegrationContextImpl.BlockCreationCallback callback) {
+		return register(group, groupedBlock, callback, ArchitectureExtensions.MOD_CONTAINER.metadata().id()); // If no id is specified, use our own id
```

Comment:
My design for datagen had all blocks under the arch-ex namespace and I want to keep that

Reply:
All blocks arch-ex makes are still in the arch-ex namespace. I feel that we're creating someone else's blocks on their behalf, so they belong in the owners' namespaces, but we can force them all into arch-ex if that's the consensus.